tests/test_jira.py

```diff
@@ -84,6 +42,46 @@ def json(self):
             }
         )
 
+    elif args == ('https://mock-org.atlassian.net/search?jql=project=TEST&startAt=0',):
+        return MockResponse(
+            {'total': 1,
```

Comment:
Let's think about whether we want to write the mocked responses into a file for the sake of readability (that would apply to all mocked responses, not only this one). I'm not sure how useful it is to have it in the code.